### DISCUSSION TITLE
How is the homebrew package repo updated, and what is the best way to get the latest version?

### DISCUSSION THREAD

**Discussion by casey** [actions/virtual-environments#6063] [General questions] [answered]
I depend on a recent update to a homebrew package, but I am still getting the old version on the macos-latest and ubuntu-latest runners. How frequently is the view of homebrew packages updated from those runners? Also, is there a way to get the latest packages? Thank you!

**Comment by al-cheb** [comment, marked as answer]
We deploy a new image once a week, if you would like to update in runtime you can use - https://docs.brew.sh/FAQ

**First update all package definitions (formulae) and Homebrew itself:**
`brew update`

**Upgrade everything with:**
`brew upgrade`

**Or upgrade a specific formula with:**
`brew upgrade <formula>`

**Reply by casey** [reply]
Nice, thank you!
